osu.Game/Modes/Scoring/Score.cs

```diff
@@ -55,10 +55,10 @@ public User User
         public long OnlineScoreID;
 
         [JsonProperty(@"username")]
-        private string temporaryUsername;
+        public string LegacyUsername;
 
         [JsonProperty(@"user_id")]
-        private long temporaryUserID;
+        public long LegacyUserID;

```

Comment:
they are only used in serialisation and since we don't yet use the jetbrains annotations there's no way to mark these exceptions without ugly `//comment disable`. public is a better solution